Samples/Client/WebClient/matrixmath.js

```diff
@@ -0,0 +1,105 @@
+
+
+function matCreate() 
```

Comment:
is this custom or did we leverage it from elsewhere?

Reply:
I started to use the JS library from http://glmatrix.net/..  but wanted something much simpler and made only methods needed for thi demo..  I did pull source for matmultiply from a portion of http://glmatrix.net/docs/mat4.js.html.  Happy to credit this source.

Reply:
ok cool - yes let's credit it, though i'm not sure exactly what the best way to do so is. per their license it looks like we should include lines 1-20 of that file (their license comment) and we can also put a link to the original source i suppose? does that sound good to you @claurendine?